boards/nordic/nrf52840dk/support/nrf52840dk_qspi_nrfutil_config.json

```diff
@@ -0,0 +1,22 @@
+{
+  "firmware_config": {
+    "peripheral": "QSPI"
+  },
+  "pins": {
+    "sck": 19,
+    "csn": 17,
+    "io0": 20,
+    "io1": 21,
+    "io2": 22,
+    "io3": 23
+  },
+  "flash_size": 67108864,
```

Comment:
This should be in bytes and not in bits, so the value should be 8388608.

Reply:
Fixed